src/review/ios/spider.js

```diff
@@ -1,9 +1,34 @@
 const rp = require('request-promise')
+const { PRESET_REGIONS } = require('../common')
 
-const getReviews = async (id) => {
-  const body = await rp(`https://itunes.apple.com/us/rss/customerreviews/page=1/sortBy=mostRecent/id=${id}/json`)
+const getReviewsByRegion = async (id, region) => {
+  const body = await rp(
+    `https://itunes.apple.com/${region}/rss/customerreviews/page=1/sortBy=mostRecent/id=${id}/json`
+  )
   const json = JSON.parse(body)
   return json.feed.entry
 }
-
-module.exports = { getReviews }
+const getReviews = id => {
+  let promises = Object.keys(PRESET_REGIONS).map(index =>
+    getReviewsByRegion(id, PRESET_REGIONS[index])
+  )
+  return Promise.all(promises).then(rawArr => {
+    let arr = []
+    for (let i in rawArr) {
+      rawArr[i] && arr.push(rawArr[i])
+    }
+    return arr
+  })
+}
+const getFlattenReviews = async id => {
+  let collections = await getReviews(id)
+  let name
+  let reviews = []
+  for (let collection of collections) {
+    if (!collection) continue
+    name = collection.shift()['im:name'].label
+    reviews = [...reviews, ...collection]
```

Comment:
If `reviews` are no longer sorted by time, you need to update `src/review/ios/util/compareReviews`

Reply:
I don't think it necessary. Since I used ``` compareReviews``` to compare the database with only one piece of data from one region rather than the one megered from four.

Reply:
There is a `break` in that method. It assumes that reviews are ordered by time.

Reply:
I read the code of `compareReviews` again and it seems that only new reviews need to be in order. So this part has no problem